beancount_import/source/schwab_csv.py

```diff
@@ -531,6 +591,44 @@ def get_narration_prefix(self) -> str:
         return "INTEREST"
 
 
+@dataclass(frozen=True)
+class FundGainsDistribution(TransactionEntry):
+    '''
+        ETFs and Mutual Funds can have distributions of capital gains
+        generated by internal activity.
+    '''
+    symbol: str
+    capital_gains_account: str
+
+    def get_sub_account(self) -> Optional[str]:
+        return "Cash"
+
+    def get_other_account(self) -> str:
+        return f"{self.capital_gains_account}:{self.symbol}"
+
+    def get_narration_prefix(self) -> str:
+        return "INCOME - CAP GAINS"
+
+    def get_postings(self) -> List[Posting]:
+        return [
+            Posting(
+                account=self.get_primary_account(),
+                units=self.amount,
+                cost=None,
+                price=None,
+                flag=None,
+                meta=self.get_meta(),
+            ),
+            Posting(
+                account=self.get_other_account(),
+                units=None,
+                cost=None,
+                price=None,
+                flag=None,
+                meta=self.get_meta(),
```

Comment:
We could introduce a `get_other_meta()` method (defaulting to `return {}` but overridden here to `return self.get_meta()`) so that you don't have to override all of `get_postings()` and repeat a lot of what is already done in the base method. It looks like all the other differences here can already be handled by existing methods e.g. `get_other_units()` and `get_cost()`.

I don't feel strongly about this though, this override is OK.

Reply:
Eh, I prefer less code so I removed the override.